src/components/Personalization/actions/setHtml.js

```diff
@@ -10,20 +10,20 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import { removePrehiding } from "../flicker";
+import { showElements } from "../flicker";
 
 export default collect => {
   return (settings, event) => {
     const { element, prehidingSelector } = event;
-    const { content, metadata } = settings;
+    const { content, meta } = settings;
 
     // this is a very naive approach, we will expand later
     element.innerHTML = content;
 
     // after rendering we should show remove the flicker control CSS classes
```

Comment:
Typo: "show remove"

Also, is the "CSS classes" part accurate/applicable still?

Reply:
@Aaronius the idea was to have something like `hideElements()` that match a particular selector and the counterpart is `showElements()` that match a selector.

If you think this is confusing, I'll try to come up with different names.